mock/py/mockbuild/podman.py

```diff
@@ -132,26 +134,38 @@ def mounted_image(self):
             subprocess.run(cmd_umount, stdout=subprocess.PIPE,
                            stderr=subprocess.PIPE, check=True)
 
-    def get_image_digest(self):
+    def get_layers_digest(self):
         """
-        Get the "sha256:..." string for the image we work with.
+        Get sha256 digest of RootFS layers. This must be identical for
+        all images containing same order of layers, thus it can be used
+        as the check that we've loaded same image.
         """
-        the_image = self.image
-        if the_image.startswith("oci-archive:"):
-            # We can't query digest from tarball directly, but note
-            # the image needs to be tagged first!
-            the_image = self._tagged_id
-        check = [self.podman_binary, "image", "inspect", the_image,
-                 "--format", "{{ .Digest }}"]
+        check = [self.podman_binary, "image", "inspect", self.image]
```

Comment:
This still needs to work with `self._tagged_id` if we work with `oci-archive:` images.